feat(alumet-relay-client): pod is now privileged and hostPID is now used (#49)
alumet-dev/helm-charts: README.md

kubectl create cm <config map name> --from-file=config=alumet-agent-client.toml
```

### Using process-to-cgroup-bridge plugin

The process-to-cgroup-bridge find the associated cgroup to a process.

> This plugin need to access PID below `/proc/`. And so`*hostPID* allow the pod to share the PID namespace, allowing the plugin to access other process executed on the node.

Reviewer: ```suggestion
This plugin needs access to PIDs below `/proc/`. The key *hostPID* allows the pod to share the PID namespace, allowing the plugin to access other processes executed on the node.
```
